liblouis/compileTranslationTable.c

```diff
@@ -2664,21 +2665,21 @@ compileRule(FileInfo *nested, CharacterClass **characterClasses,
 						nested, ruleChars.chars[0], ruleDots.chars[0], displayTable);
 			}
 		break;
+	case CTO_IncludeFile: {
```

Comment:
I don't understand why the `CTO_IncludeFile` case was in the `default:` before and now stands on its own as a separate case

Reply:
This was just a mistake in the previous commit (c322368b911fe92eb6b05d45f7bbbc2b317dd4dc). I moved a bunch of cases inside a nested switch in order to be able to insert the `if (!table) break;`.

Reply:
OK